srcpkgs/soju/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'soju'
+pkgname=soju
+version=0.2.1
+revision=1
+build_style=gnu-makefile
```

Comment:
```suggestion
build_style=go
```
Remove do_build and add installing the config and manpage to post_install.

Reply:
Thanks for the comment.
Is it usually how go packages are built, even if developers provide a Makefile?
Does this mean I should ignore the flags recommend [here](https://wiki.archlinux.org/title/Go_package_guidelines) for packaging go applications?

Reply:
> Is it usually how go packages are built, even if developers provide a Makefile?

If you use the makefile, you will have to make sure it cross compiles properly and applies the correct flags which is not worth the added complexity. Not using the makefile is better, but not ideal. Some projects have a target in the makefile which doesn't touch the go part and installs only the config file, manpage etc.

> Does this mean I should ignore the flags recommend here for packaging go applications?

That's a guide for Arch Linux, I don't see why this is relevant here. The go build style will apply the flags we want.